drivers/espi/Kconfig.npcx

```diff
@@ -48,6 +48,14 @@ config ESPI_NPCX_PERIPHERAL_DEBUG_PORT_80_MULTI_BYTE
 	  EC can accept 1/2/4 bytes of Port 80 data written from the Host in an
 	  eSPI transaction.
 
+config ESPI_NPCX_PERIPHERAL_DEBUG_PORT_80_RING_BUF_SIZE
+	int "Debug Port80 ring buffer size"
+	depends on SOC_SERIES_NPCX9 && ESPI_PERIPHERAL_DEBUG_PORT_80
```

Comment:
I think that this depends and the one above should match the build config that enables the actual driver to be built + the condition in the file, so that would be `depends on ESPI_NPCX && ESPI_PERIPHERAL_DEBUG_PORT_80`, which means you could move it under the "if ESPI_NPCX" below and keep this a bit simpler.

Reply:
`ESPI_NPCX_PERIPHERAL_DEBUG_PORT_80_MULTI_BYTE` is only supported in the NPCX9 chip series.
`ESPI_NPCX_PERIPHERAL_DEBUG_PORT_80_RING_BUF_SIZE` is used when `ESPI_NPCX_PERIPHERAL_DEBUG_PORT_80_MULTI_BYTE` is defined.
In the latest patch, I change `ESPI_NPCX_PERIPHERAL_DEBUG_PORT_80_RING_BUF_SIZE` to depend on `ESPI_NPCX_PERIPHERAL_DEBUG_PORT_80_MULTI_BYTE`.
I also change some codes related to the multiple support to be guarded by 'CONFIG_ESPI_NPCX_PERIPHERAL_DEBUG_PORT_80_MULTI_BYTE。